source/bdDetect.py

```diff
@@ -26,28 +26,43 @@
 from baseObject import AutoPropertyObject
 import re
 from winAPI import messageWindow
+import extensionPoints
 
 
 HID_USAGE_PAGE_BRAILLE = 0x41
 
-
 DBT_DEVNODES_CHANGED=7
 
 _driverDevices = OrderedDict()
 USB_ID_REGEX = re.compile(r"^VID_[0-9A-F]{4}&PID_[0-9A-F]{4}$", re.U)
 
-class DeviceMatch(
-	namedtuple("DeviceMatch", ("type","id", "port", "deviceInfo"))
-):
+
+class DeviceMatch(typing.NamedTuple):

```

Comment:
would you be able to change new types added to `from typing import ...` instead of `typing.` for brevity?